aspnetcore/grpc/performance.md

```diff
@@ -274,10 +274,12 @@ gRPC is a message-based RPC framework, which means:
 * The entire message is loaded into memory before gRPC can send it.
 * When the message is received, the entire message is deserialized into memory.
 
-Messages with large binary payloads can allocate byte arrays on the [large object heap](/dotnet/standard/garbage-collection/large-object-heap). Large allocations impact server performance and scalability.
+Binary payloads are allocated as a byte array. For example, a 10 megabyte binary payload allocates a 10 megabyte byte array. Messages with large binary payloads can allocate byte arrays on the [large object heap](/dotnet/standard/garbage-collection/large-object-heap). Large allocations impact server performance and scalability.
```

Comment:
```suggestion
Binary payloads are allocated as a byte array. For example, a 10 MB binary payload allocates a 10 MB byte array. Messages with large binary payloads can allocate byte arrays on the [large object heap](/dotnet/standard/garbage-collection/large-object-heap). Large allocations impact server performance and scalability.
```

Reply:
Or ` .. a 10 megabyte (MB) ... ` on first usage then use MB on the next occurrence.  We use MB by itself extensively and have no reported problems with that.